freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -239,6 +247,28 @@ private Security buildSecurity(InstanceGroup ig) {
         return new Security(rules, ig.getSecurityGroup().getSecurityGroupIds(), true);
     }
 
+    private Optional<CloudFileSystemView> buildFileSystemView(Stack stack) {
+        Telemetry telemetry = stack.getTelemetry();
+        if (telemetry != null && telemetry.getLogging() != null) {
+            Logging logging = telemetry.getLogging();
+            if (logging.getStorageLocation() != null) {
+                if (logging.getS3() != null) {
+                    CloudS3View s3View = new CloudS3View(CloudIdentityType.LOG);
+                    s3View.setInstanceProfile(logging.getS3().getInstanceProfile());
+                    return Optional.of(s3View);
+                } else if (logging.getAdlsGen2() != null) {
+                    CloudAdlsGen2View adlsGen2View = new CloudAdlsGen2View(CloudIdentityType.LOG);
+                    adlsGen2View.setAccountKey(logging.getAdlsGen2().getAccountKey());
```

Comment:
Minor: extract `logging.getAdlsGen2()` as a local variable and replace all below occurrences.